src/XMakeTasks/UnitTests/AssignTargetPath_Tests.cs

```diff
@@ -58,7 +58,9 @@ public void OutOfCone()
                               new TaskItem(
                                   NativeMethodsShared.IsWindows ? @"d:\f1\f2\f3\f4\file.txt" : "/f1/f2/f3/f4/file.txt")
                           };
-            t.RootFolder = NativeMethodsShared.IsWindows ? @"c:\f1" : "/f1";
+            // Note that on Windows we change the drive to make sure the file is out of the cone.
+            // If not Windows, where there is no drive, we change the root directory from f1 and x1
```

Comment:
Note that on Windows it goes from d:\f1 to c:\f1. Must be out of the cone. On Unix we move it out of the cone by changing the name of the root from f1 to x1.